pkg/cmd/attestation/verification/extensions.go

```diff
@@ -34,28 +34,23 @@ func VerifyCertExtensions(results []*AttestationProcessingResult, ec Enforcement
 	return lastErr
 }
 
-func verifyCertExtensions(verifiedCert certificate.Summary, criteria EnforcementCriteria) error {
-	sourceRepositoryOwnerURI := verifiedCert.Extensions.SourceRepositoryOwnerURI
-	if !strings.EqualFold(criteria.Certificate.SourceRepositoryOwnerURI, sourceRepositoryOwnerURI) {
-		return fmt.Errorf("expected SourceRepositoryOwnerURI to be %s, got %s", criteria.Certificate.SourceRepositoryOwnerURI, sourceRepositoryOwnerURI)
+func verifyCertExtensions(verified, expected certificate.Summary) error {
```

Comment:
nice

Reply:
i understand that in context this is from a verified certificate, but it's a bit confusing given the name of the func.

what if, idk,

```suggestion
func verifyCertExtensions(given, expected certificate.Summary) error {
```

Reply:
Yeah, that makes sense. `verified` is confusing within the function itself.